.github/workflows/php.yml

```
@@ -2,9 +2,14 @@ name: PHP Composer
 
 on:
   push:
-    branches: [ main ]
+    branches:
+      - 'main'
+      - '**/**'
```

What is the goal here?  Why not just

```suggestion
    branches:
      - '**'
```